### DISCUSSION TITLE
Assembly of the T650

### DISCUSSION THREAD

**Discussion by MaximeBussios** [ctu-mrs/mrs_uav_system#101] [Q&A] [answered]
Hello MRS-team,

I'm currently building the T650 and I was wondering if you had a CAD file or a piece of documentation you would accept to share to help me place every equipments in a smart way. It would save me a lot of time !

Thank you for your help,
Maxime Bussios.

**Comment by matemat13** [comment, marked as answer]
@DanHert, @stoudpav?

**Reply by MaximeBussios** [reply]
I've found one in the end ( https://grabcad.com/library/tarot-ironman-650-2 ) but thanks !
